lib/src/body.rs

```diff
@@ -1,5 +1,8 @@
 //! Body type, for request and response bodies.
 
+use crate::streaming_body::StreamingBodyItem;
+use crate::Error;
+
 use {
     crate::error,
```

Comment:
sorry, nit: can we squish this into the `use` below?

Reply:
Done in 5f2d111. Do you know if there's a combination of settings that would get `rust-analyzer` to insert imports this way? https://rust-analyzer.github.io/manual.html#auto-import 

I tend to exclusively use the auto-import feature without even looking at what it inserts, which is how I missed this inconsistency.